streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -81,8 +80,12 @@
                     new MockStateStoreSupplier(storeName1, false).get(),
                     new MockStateStoreSupplier(storeName2, true).get()
             ),
-            Collections.<String, String>emptyMap(),
-            Collections.<StateStore, ProcessorNode>emptyMap());
+            new HashMap<String, String>() {
+                {
+                    put(storeName1, storeChangelogTopicName1);
+                    put(storeName2, storeChangelogTopicName2);
+                }
+            });
```

Comment:
Yes. But in the original version of this test we pass in two empty maps (`sourceStoreToSourceTopics` and `storeToProcessorNodeMap` -- so I am wondering why we do need to provide this information now but not before (or where did this information come from before). Still confused.